lib/vm/src/export.rs

```diff
@@ -38,6 +39,9 @@ pub struct ExportFunction {
     /// Address of the function call trampoline owned by the same VMContext that owns the VMFunctionBody.
     /// May be None when the function is a host-function (FunctionType == Dynamic or vmctx == nullptr).
     pub call_trampoline: Option<VMTrampoline>,
+    /// A reference to the instance, to ensure it's not deallocated
+    /// before the function.
+    pub instance_allocator: Option<Arc<InstanceAllocator>>,
```

Comment:
You mentioned it on Slack, but it'd be good to explain the semantics of the `Option` in these doc comments

Reply:
Correct, I'll!